save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/LnkUserNotification.kt

```diff
@@ -0,0 +1,26 @@
+package com.saveourtool.save.entities
+
+import com.saveourtool.save.spring.entity.BaseEntity
+import javax.persistence.Entity
+import javax.persistence.JoinColumn
+import javax.persistence.ManyToOne
+
+/**
+ * @property notification
+ * @property user
+ * @property isShow
+ */
+@Entity
+class LnkUserNotification(
```

Comment:
do we need it? maybe add `user` and `isShow` to `Notification`?
I understand that you try to avoid duplicates in notification, but I think it can complicate the logic

Reply:
Yes, I wanted to get rid of duplicates, but let it be as you suggest.